### ISSUE TITLE
Add hierarchy to the year facet

### ISSUE THREAD

**Issue by thatbudakguy** [sul-dlss/earthworks#1136]
The blacklight_range_limit slider has accessibility issues. We're looking at implementing a hierarchical facet, similar to vt-arclight:

![Screenshot 2024-08-08 at 09 45 59](https://github.com/user-attachments/assets/d62907d0-66e6-4a48-9270-bddb15f4c0c4)

To get started, see [the implementation there](https://github.com/sul-dlss/vt-arclight/blob/8206bfb9f2fb1c4b52a8682d6efbd97d0a579b0d/app/controllers/catalog_controller.rb#L136-L140).
